akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -697,14 +697,52 @@ object Source {
    * for downstream demand unless there is another message waiting for downstream demand, in that case
    * offer result will be completed according to the overflow strategy.
    *
-   * SourceQueue that current source is materialized to is for single thread usage only.
+   * Current source is materialized for the single producer usage only.
```

Comment:
The materialized `SourceQueue` may only be used from a single producer.

Reply:
Applied